docs/aerodromes/classc/Brisbane.md

```diff
@@ -29,6 +29,74 @@ There are three SMC positions at Brisbane which are responsible for various part
   <figcaption>YBBN Maneuvering Area Responsibility</figcaption>
 </figure>
 
+## Helicopter Operations
+Brisbane has one helipad located within the general aviation apron, most rescue helicopters are based within the general aviation maintenance area and will depart from the **H2 taxiway** to save time when operating.
+
+#### Waypoints
+VFR and IFR helicopters are normally processed via two visual waypoints north and south of the field, when in VMC conditions. IFR helicopters in IMC conditions may conform to fixed wing ops and be processed via the BN (RADAR) SID from an appropriate runway, unless a visual departure is acceptable. These waypoints are used for **departures** and **arrivals**, depending on weather and traffic conditions.
+
+
+| Direction               | Waypoint       | Name        |
+| ------------------ | -------------- | ---------------- |
+| North      | BLHS   | Bald Hills          |
+| South      | MBHM   | Manly Boat Harbour          |
+| East      | MBHM   | Manly Boat Harbour          |
+| West      | BLHS   | Bald Hills          |
+
+#### Departures
+
+Refer to [Maneuvering Area Responsibility](#maneuvering-area-responsibility) to determine which SMC or ADC position (when non-standard positions are online) is responsible for managing helicopter arrivals and departures.
+
+!!! example
+    **RSCU533:** "Brisbane Delivery, helicopter RSCU533, medivac for YXHE, request clearance"  
+    **BN ACD:** "RSCU533, Brisbane Delivery, cleared to YXHE via BLHS, climb 2500, squawk 1256, departure frequency 124.7"  
+    **RSCU533:** "Cleared to YXHE via BLHS, climb 2500, squawk 1256, departures 124.7, RSCU533"
+
+Taxiway H2 is inside the maneuvering area and therefore owned by **BN SMC-N**, treated like a runway requiring a takeoff and landing clearance by BN ADC. Helicopters departing on other surfaces outside the maneuvering areas should be instructed to *"report airborne"* so the controllers knows when they are active in their airspace. 
+
+!!! example
+    **RSCU533:** "Brisbane Tower, helicopter RSCU533, H2 ready"  
+    **BN ADC:** "RSCU533, Brisbane Tower, H2 cleared for takeoff"  
+    **RSCU533:** "H2 cleared for takeoff, RSCU533"
+ 
+    **BN ADC:** "RSCU533, contact departures"  
+    **RSCU533:** "Departures, RSCU533"
+
+#### Arrivals
+
+IFR helicopters operating within IMC conditions may conform to fixed wing ops and be processed via an instrument approach for a appropriate duty runway. Standard fixed wing phraseology shall be used during these conditions.
+
+!!! example
+    **RSCU533:** "Brisbane Tower, helicopter RSCU533"  
+    **BN ADC:** "RSCU533, Brisbane Tower, cleared visual approach H2"  
+    **RSCU533:** "Cleared visual approach H2, RSCU533"
```

Comment:
We'll need to sort out the coordination required if ADC is going to clear the aircraft for a visual approach. Might be easier to give BN ADC ownership of the CTR

Reply:
Not sure if it's worth mentioning, but IRL ADC (120.5) often clears aircraft landing on H2 for the visual approach, and pretty sure ADC-N (118.0) clear aircraft for the visual approach F4 HLS.

Reply:
Yeah cool, I guess that makes sense based on their likely inbound direction.